src/registry/registry.json

```diff
@@ -2917,7 +2949,8 @@
     "managedtopic": "managedtopics",
     "sharingcriteriarule": "sharingrules",
     "botversion": "bot",
-    "customfieldtranslation": "customobjecttranslation"
+    "customfieldtranslation": "customobjecttranslation",
+    "datamappingfielddefinition": "datamappingobjectdefinition"
   },
-  "apiVersion": "53.0"
+  "apiVersion": "54.0"
```

Comment:
no, this should be left as the current api version... there are certain scenarios where this will be used when making the API call...I don't _think_ that there will be any issues by having your type in the registry now, even if it isn't supported until 54

Reply:
@WillieRuemmele , reverted the change.